lib/Form/Label/index.js

```diff
@@ -28,11 +30,24 @@ const Label = ({
     }
   };
 
+  const labelIsHovered = () => {
+    if (highlight) {
+      highlightIsHovered();
```

Comment:
Why do we need to tell if we're hovering over the highlight? Doesn't this cause a loop as the parent set states...causing a re-render for all children and then we tell the parent to set the state here. Is that an issue?

Reply:
It might be that we could make this simpler by saying;

- labels (buttons in this case) execute a function on mouse enter&leave
- If groups need to know when we're hovering over it then maybe the developer could handle that in the consuming app? E.g. passing a `highlighted` prop down to the group. The same could be said about being `active`...allow a active prop to be passed down.

By simplifying it like so it'll make the UI component easier to use and will remove the complexities around why we need `highlightIsHovered`, `highlightIsNotHovered` and the previous comment on this thread.

What do you think?